paper/paper.md

```diff
@@ -11,36 +11,57 @@ authors:
+
+# Implementation Overview
+
+ERAD is a free, open-source Python toolkit for computing equity and resilience measures in the face of hazards like earthquakes and flooding. It uses a graph database to store data and perform computation down to the household level for a variety of critical services that are connected by the electric grid, with an initial focus on the local, distribution system. 
 
-In last few years, we have seen surge in disaster events affecting power system grid thanks to climate change. Lot of the research and software tools developed for understanding grid resilience have been focused on power transmission system. Recently we have seen increased intetrest in understanding equity and resilience in distribution power system. ERAD is developed and made publicy available to understand distribution grid resilience and equity impacts due to hazard events. You can also evaluate effectiveness of different post disaster resilience and equity improvement programs using this software.
+Users provide*<u>.... XXX (Complete this paragraph. Here is a start)</u>* geospatial data on the location of customers and grid infrastructure along with their connectivity, such as from an OpenDSS [@opendss] model. It also uses critical infrastructure location data, such as from XXX, and demographic data, such as from YYY. This enables ERAD to compute user-configurable baseline social burden-style metrics for each household's access to the suite of services as a function of distance or other ease of access metric. ERAD hazard models then simulate a range of disaster events and their damage to grid equipment and corresponding outages to households and critical services providers alike--unless there is a local electricity supply such as on-site backup storage or microgrids that can keep intentional islands of the grid energized. For scalability this evaluation is not based connectivity and supplyu/demand balance for each potentially isolated electrical island, rather than the more time-consuming power flow analysis used in engineering studies. The same service access metrics can then be computed for the disaster scenario to identify both the absolute level of service access during a disaster and the relative impacts compared to pre-disaster. ERAD also provides user-configurable methods to computed aggregated resiliency metrics and distributions at the community level (e.g. for each neighborhood) and to evaluate correlations with various demographics to assess equity in service access and resilience.
```

Comment:
I took a stab at this. It might be close but some of the pieces need to be filled out